### PR TITLE
Add some information about local development to README

### PR DESCRIPTION
**Pull request by hzsweers**
Based on my spelunking through the codebase today

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,27 @@ We'd love for you to contribute to our open source projects. Before we can accep
 - If you **have a feature request**, open an issue or submit an implementation via a pull request
 - If you **want to contribute**, submit a pull request.
 
+### Development
+
```
Comment (kageiit) [inline README.md]: Please move this to contributing.md

Comment (hzsweers) [reply README.md]: Contributing is information to show before filing an issue or PR, which is after local development is started. Most things like this are put into a README section or wiki page

**Comment by kageiit** [comment]
Let's move to a new developing wiki page then. This information is not suitable for readme

**Comment by hzsweers** [comment]
Sure. Do you have any comments on this bit though? (Can't PR wiki changes)

**Comment by hzsweers** [comment]
Sidenote - by that logic, shouldn't the code style section above it also be moved? That was the main reason I figured this project followed conventions with having local development info in the README

**Pull request merged**